site/documentation/content/api/Metrics.md

```diff
@@ -108,4 +109,4 @@ For an incoming message of size 10KB, it is reported as 12KB.
 
 ### Service Metrics
 
-Hono's service components do not report any metrics at the moment.
+Hono's service components do not report own metrics at the moment.
```

Comment:
IMHO we should also mention that the Command Router component reports *hono.commands.received* and *hono.commands.payload* ...

Reply:
I've added a sentence here on that.